Octokit.Tests/Clients/RepositoriesClientTests.cs

```diff
@@ -771,7 +771,7 @@ public void GetsCorrectUrl()
                 client.Get("owner", "name", "reference");
 
                 connection.Received()
-                    .Get<GitHubCommit>(Arg.Is<Uri>(u => u.ToString() == "repos/owner/name/commits/reference"));
+				          .Get<GitHubCommit>(Arg.Is<Uri>(u => u.ToString() == "repos/owner/name/commits/reference"), null, AcceptHeaders.SignatureVerificationPreview);
```

Comment:
Also, we don't want to use implementation logic as part of the test, so rather than using the `AcceptHeaders` helper, can you hardcode the expected header in the tests?